docs/pages/database-access/enroll-aws-databases/redis-aws.mdx

```diff
@@ -38,22 +43,50 @@ description: How to configure Teleport database access with AWS ElastiCache and
 
 (!docs/pages/includes/database-access/alternative-methods-join.mdx!)
 
-<Tabs>
-
-<TabItem label="ElastiCache">
-
-(!docs/pages/includes/database-access/aws-create-db-configuration.mdx type="ElastiCache" matcher="elasticache" protocol="redis" example="clustercfg.my-redis-cluster.xxxxxx.usw1.cache.amazonaws.com:6379" !)
+Install Teleport on the host where you will run the Teleport Database Service:
 
-</TabItem>
-
-<TabItem label="MemoryDB">
+(!docs/pages/includes/install-linux.mdx!)
 
-(!docs/pages/includes/database-access/aws-create-db-configuration.mdx type="MemoryDB" matcher="memorydb" protocol="redis" example="clustercfg.my-memorydb.xxxxxx.memorydb.us-west-1.amazonaws.com:6379" !)
-
-</TabItem>
+Create the Database Service configuration:
 
+<Tabs>
+  <TabItem label="ElastiCache">
+
+  Change `example.teleport.sh:443` to the host and port of your Teleport Proxy
+  Service. Set `ELASTICACHE_URI` to the domain name and port of your ElastiCache
+  database:
+
+  ```code
+  $ ELASTICACHE_URI=""
+  $ sudo teleport db configure create \
+     -o file \
+     --proxy=example.teleport.sh:443 \
+     --protocol="redis" \
+     --uri=${ELASTICACHE_URI?} \
```

Comment:
```suggestion
     --uri=${ELASTICACHE_URI} \
```
?

Reply:
This instructs the shell to raise an error if the variable isn't defined